runtimes/eden/src/xcm_config.rs

```diff
@@ -0,0 +1,153 @@
+use super::{
+	AccountId, Balance, Balances, Call, Event, Origin, ParachainInfo, ParachainSystem, PolkadotXcm, Runtime, XcmpQueue,
+};
+use crate::{implementations::DealWithFees, pallets_governance::EnsureRootOrMoreThanHalfOfTechComm};
```

Comment:
Can you please remove `pallets_governance::EnsureRootOrMoreThanHalfOfTechComm` as suggested by clippy.

Reply:
done